src/components/Book.js

```diff
@@ -11,10 +11,20 @@ const Book = (props) => {
   };
 
   return (
-    <div>
-      <h3>{book.title}</h3>
-      <p>{book.author}</p>
-      <button type="button" onClick={() => deleteBook(book.id)}>Remove</button>
+    <div className="bookItem">
+      <div className="book">
+        <h5 className="category">Inspirational</h5>
+        <h1 className="bookTitle">{book.title}</h1>
+        <p className="bookAuthor">{book.author}</p>
+        <button type="button" className="onBook comment">Comment</button>
+        <button type="button" onClick={() => deleteBook(book.id)} className="remove">Remove</button>
+        <button type="button" className="onBook">Edit</button>
+      </div>
+      <div className="chapterUpdate">
+        <p className="chapterHeading">CURRENT CHAPTER</p>
+        <p className="chapter">Chapter 1</p>
+        <button type="button" className="progress">UPDATE PROGRESS</button>
+      </div>
```

Comment:
- [ ] To follow the design please add a progress circle.

![image](https://user-images.githubusercontent.com/38635158/167259341-501c0bb3-59d4-4ec0-b09f-444ed2262823.png)